### PR TITLE
Updated #733, #742 postman test cases

### PR DESCRIPTION
**Pull request by amarmodus**
Updated #733, #742 postman test cases in Golden_Path.postman_collection.json
created framework for #956 test case too

### CODE REVIEW

File: Golden_Path.postman_collection.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
 	"info": {
-		"_postman_id": "5591457e-465c-43be-bd9d-4863ebd7747e",
-		"name": "Golden_Path",
+		"_postman_id": "6853006f-5da9-47e9-8c5b-f54b5e1bd15c",
+		"name": "Golden_Path Copy",
```
Comment (elnyry) [inline Golden_Path.postman_collection.json]: Copy?

File: Golden_Path.postman_collection.json [hunk 2]
```diff
@@ -204,6 +204,7 @@
 							}
 						}
 					],
+					"protocolProfileBehavior": {},
```
Comment (elnyry) [inline Golden_Path.postman_collection.json]: what does this mean? lets discuss

File: Golden_Path.postman_collection.json (hunk 1 above)
Comment (amarmodus) [reply Golden_Path.postman_collection.json]: Updated default file name in 2nd commit - exporting issue.

File: Golden_Path.postman_collection.json (hunk 2 above)
Comment (amarmodus) [reply Golden_Path.postman_collection.json]: This is automatically added by Postman and no change to functionality as properties are empty.

**Comment by elnyry** [comment]
Tested, had Golden path 464:0

**Pull request merged**